demoapp/src/androidTest/java/me/uport/sdk/demoapp/PersonalSignatureTest.kt

```diff
@@ -11,21 +11,27 @@ import android.support.test.espresso.assertion.ViewAssertions.matches
 import android.support.test.espresso.intent.Intents.intending
 import android.support.test.espresso.intent.matcher.IntentMatchers.hasAction
 import android.support.test.espresso.intent.rule.IntentsTestRule
+import android.support.test.espresso.matcher.ViewMatchers.isDisplayed
 import android.support.test.espresso.matcher.ViewMatchers.withId
 import android.support.test.espresso.matcher.ViewMatchers.withText
 import me.uport.sdk.demoapp.request_flows.PersonalSignRequestActivity
 import me.uport.sdk.transport.RequestDispatchActivity
+import org.hamcrest.CoreMatchers.allOf
 import org.hamcrest.CoreMatchers.not
 import org.junit.After
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
+import ro.mirceanistor.testutil.ViewMatcherIdlingRule
 
 class PersonalSignatureTest {
 
     @get:Rule
     val intentsTestRule = IntentsTestRule(PersonalSignRequestActivity::class.java)
 
+    @get:Rule
+    val viewMatcherIdlingRule = ViewMatcherIdlingRule(allOf(withId(R.id.progress), isDisplayed()))
```

Comment:
viewMatcherIdlingRule doesn't seem to ever be used. Is it necessary to have a variable for this?